pages/login.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <p>style/login</p>
```

Comment:
이 브랜치는 머지 진행해주시고 아래 기능 추가 부탁드립니다.
```suggestion
   <p>style/login - 기능 추가</p>
```